uPortal-webapp/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -656,6 +656,7 @@
           <xsl:attribute name="class">
             <xsl:text>up dashboard portal fl-theme-mist</xsl:text>
             <xsl:if test="$PORTAL_VIEW='focused'"> up-focused</xsl:if>
+            <xsl:if test="$AUTHENTICATED!='true'"> guest</xsl:if>
```

Comment:
Where possible use the `up` prefix for custom css.
`guest` -> `up-guest`

Reply:
done !